apps/frontend/src/components/cards/controltable/ControlRowHeader.vue

```diff
@@ -39,25 +39,46 @@
       </v-row>
     </template>
 
+    <template #impact>
+      <v-card-text class="pa-2">
+        <v-chip active-class="NONE" outlined :color="impact_color">
+          {{ control.data.impact * 10 }}
+        </v-chip>
+      </v-card-text>
+    </template>
+
     <template #severity>
       <v-card-text class="pa-2">
-        <div v-if="showImpact">
-          <CircleRating
-            :filled-count="severity_arrow_count(control.hdf.severity)"
-            :total-count="4"
-          />
-          <v-divider class="mx-1" />
-          {{ (control.hdf.severity || 'none').toUpperCase() }}
-        </div>
-        <div v-else>
-          <CircleRating
-            :filled-count="severity_arrow_count(control.data.tags.severity)"
-            :total-count="4"
-          />
-          <br />
-          <v-divider class="mx-1" />
-          {{ (control.data.tags.severity || 'none').toUpperCase() }}
-        </div>
+        <v-tooltip v-if="'severityoverride' in control.data.tags" bottom>
```

Comment:
Might be worthwhile updating the styling for v-tooltip so that the background is opaque.  The translucent background sometimes makes it hard to read text in the tooltip that is overlaid on top of other text on the page.  An example that's happening right now is the tooltip on the severity override overlapping the title text.

This will probably need to be fixed application wide.

Reply:
Done